### PR TITLE
fix(Core/Spells): Flame Wave

### PR DESCRIPTION
**Pull request by avarishd**
## Changes Proposed:

This PR proposes changes to:
-  [ ] Core (units, players, creatures, game systems).
-  [x] Scripts (bosses, spell scripts, creature scripts).
-  [x] Database (SAI, creatures, etc).

## Issues Addressed:

- Closes https://github.com/chromiecraft/chromiecraft/issues/4718
- Closes https://github.com/chromiecraft/chromiecraft/issues/4678
- Closes https://github.com/azerothcore/azerothcore-wotlk/issues/14397

## SOURCE:

The changes have been validated through:
- [ ] Live research (checked on live servers, e.g Classic WotLK, Retail, etc.)
- [ ] Sniffs (remember to share them with the open source community!)
- [x] Video evidence, knowledge databases or other public sources (e.g forums, Wowhead, etc.)
- [x] The changes promoted by this pull request come partially or entirely from another project (cherry-pick). **Cherry-picks must be committed using the proper --author tag in order to be accepted, thus crediting the original authors, unless otherwise unable to be found**

Faction & timers from TC/Mangos-wotlk DB
faction ID cherry-pick 

-- https://github.com/TrinityCore/TrinityCore/commit/e3dcf158629214ab84699d76ebeab3c14222ed74
-- Co-Authored-By: Carlos Morales <carlos.morales@insmet.cu>

## Tests Performed:

This PR has been:
- [x] Tested in-game by the author.
- [ ] Tested in-game by other community members/someone else other than the author/has been live on production servers.
- [ ] This pull request requires further testing and may have edge cases to be tested.

## How to Test the Changes:

- [x] This pull request can be tested by following the reproduction steps provided in the linked issue
- [ ] This pull request requires further testing. Provide steps to test your changes. If it requires any specific setup e.g multiple players please specify it as well.

.go c id 18975/19740

## Known Issues and TODO List:

- [ ] [Flame Wave 38611](https://wowgaming.altervista.org/aowow/?spell=38611) probably requires the same correction
- [x] Overlapping area auras deals multiple instances of damage
image

## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.

### CODE REVIEW

**Comment by Kitzunu** [comment]
Conflict

File: data/sql/updates/pending_db_world/rev_1719208787140826600.sql [hunk 1]
```diff
@@ -0,0 +1,7 @@
+-- Flame Wave.
+DELETE FROM `smart_scripts` WHERE (`source_type` = 0 AND `entryorguid` IN (18975,19740));
+INSERT INTO `smart_scripts` (`entryorguid`, `source_type`, `id`, `link`, `event_type`, `event_phase_mask`, `event_chance`, `event_flags`, `event_param1`, `event_param2`, `event_param3`, `event_param4`, `event_param5`, `event_param6`, `action_type`, `action_param1`, `action_param2`, `action_param3`, `action_param4`, `action_param5`, `action_param6`, `target_type`, `target_param1`, `target_param2`, `target_param3`, `target_param4`, `target_x`, `target_y`, `target_z`, `target_o`, `comment`) VALUES
+(18975, 0, 0, 0, 0, 0, 75, 0, 5000, 10000, 15000, 20000, 0, 0, 11, 33804, 0, 0, 0, 0, 0, 2, 0, 0, 0, 0, 0, 0, 0, 0, 'Wrathguard - In Combat - Cast \'Flame Wave\''),
+(19740, 0, 0, 0, 0, 0, 75, 0, 5000, 10000, 15000, 20000, 0, 0, 11, 33804, 0, 0, 0, 0, 0, 2, 0, 0, 0, 0, 0, 0, 0, 0, 'Wrathwalker - In Combat - Cast \'Flame Wave\'');
+
+UPDATE `creature_template` SET `faction` = 90 WHERE (`entry` = 19381);
```
[the remark this answers is not in the document]
Comment (Nyeriah) [reply data/sql/updates/pending_db_world/rev_1719208787140826600.sql:7]: @avarishd

[the remark this answers is not in the document]
Comment (avarishd) [reply data/sql/updates/pending_db_world/rev_1719208787140826600.sql:7]: sniff what? the timers? idk

[the remark this answers is not in the document]
Comment (Nyeriah) [reply data/sql/updates/pending_db_world/rev_1719208787140826600.sql:7]: The faction he means

[the remark this answers is not in the document]
Comment (avarishd) [reply data/sql/updates/pending_db_world/rev_1719208787140826600.sql:7]: image
timers = Amplitude

**Review by Nyeriah — APPROVED** [review]

**Comment by Nyeriah** [comment]
Please fix the merge conflict. Otherwise seems ready to go for me

**Comment by Nyeriah** [comment]
For some reason its still saying there are conflicts to be resolved

**Pull request merged**